tests/main/econnreset/task.yaml

```diff
@@ -9,18 +9,25 @@ restore: |
 # with the distro
 backends: [-autopkgtest]
 
+debug: |
+    echo "Partial download status"
+    ls -lh test-snapd-huge_* || true
+    echo "other dir content"
+    ls -lh
```

Comment:
Would it make sense to dump /var/lib/snapd/cache here and reset it before the test?

Reply:
I mean, maybe? But `snap download` does not use that directory.

Reply:
Ah, never mind then.